src/content/nl/2019/accessibility.md

```diff
@@ -0,0 +1,320 @@
+
+### Koppen
+
+Koppen zijn niet alleen visueel nuttig, maar ook voor schermlezers. Ze stellen schermlezers in staat snel van sectie naar sectie te springen en helpen aan te geven waar de ene sectie eindigt en de andere begint.
+
+Om verwarring van gebruikers van schermlezers te voorkomen, moet u ervoor zorgen dat u nooit een kopniveau overslaat. Ga bijvoorbeeld niet rechtstreeks van een `H1` naar een `H3` en sla de `H2` over. Waarom is dit zo belangrijk? Omdat dit een onverwachte wijziging is, waardoor een gebruiker van een schermlezer denkt dat hij een stukje inhoud heeft gemist. Dit kan ertoe leiden dat ze helemaal opnieuw gaan zoeken naar wat ze hebben gemist, zelfs als er niets ontbreekt. Bovendien help je al je lezers door een consistenter ontwerp te behouden.
+
+Met dat gezegd zijnde, hier zijn onze resultaten:
+
+1. 89,36% van de pagina's gebruikt op de een of andere manier koppen. Geweldig.
+2. 38,6% van de pagina's slaat kopniveaus over.
+3. Vreemd genoeg worden `H2`'s op meer sites gevonden dan `H1`'s.
+
+{{ figure_markup(
+  image="fig3.png",
+  caption="Populariteit van kopniveaus.",
+  description="Verticaal staafdiagram met percentagegegevens, variërend van 0 tot 80 in stappen van 20, versus balken die elk kopniveau h1 tot en met h6 vertegenwoordigen. `H1`: 63,25%; `H2`: 67,86%; `H3`: 58,63%; `H4`: 36,38%; `H5`: 14,64%; `H6`: 6,91%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=1123601243&format=interactive"
+  )
+}}
+
+### Hoofdinhoudsgebied
+
+Een [Hoofdinhoudsgebied <i lang="en">`main`</i>](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role) geeft aan schermlezers aan waar de hoofdinhoud van een webpagina begint, zodat gebruikers er direct naartoe kunnen springen. Zonder dit moeten gebruikers van schermlezers uw navigatie handmatig overslaan elke keer dat ze naar een nieuwe pagina op uw site gaan. Dit is natuurlijk nogal frustrerend.
+
+We ontdekten dat slechts één op de vier pagina's (26,03%) een belangrijk Hoofdinhoudsgebied bevatte. En verrassend genoeg bevatte 8,06% van de pagina's ten onrechte meer dan één belangrijk Hoofdinhoudsgebied, waardoor deze gebruikers raden welk gebied de werkelijke hoofdinhoud bevat.
+
+{{ figure_markup(
+  image="fig4.png",
+  caption="Percentage pagina's op basis van hun aantal `main` elementen.",
+  description="Verticaal staafdiagram met percentagegegevens, variërend van 0 tot 80 in stappen van 20, versus balken die het aantal "Hoofd" inhoudsgebied per pagina van 0 tot 4 weergeven. Bron: HTTP Archive (juli 2019). Nul: 73,97%; Een: 17,97%; Twee: 7,41%; Drie: 0,15%; 4: 0,06%.",
```

Comment:
Can't have quotes within quotes and think this should be `main` anyway - could you correct this in English version and add to this PR?

```suggestion
  description="Verticaal staafdiagram met percentagegegevens, variërend van 0 tot 80 in stappen van 20, versus balken die het aantal `main` inhoudsgebied per pagina van 0 tot 4 weergeven. Bron: HTTP Archive (juli 2019). Nul: 73,97%; Een: 17,97%; Twee: 7,41%; Drie: 0,15%; 4: 0,06%.",
```

Reply:
Going to accept this one to allow the tests to run.